backend/stacks/persistent_stack/admin_users.py

```diff
@@ -0,0 +1,62 @@
+from aws_cdk.aws_cognito import ResourceServerScope, OAuthScope
+from aws_cdk.aws_kms import IKey
+from constructs import Construct
+
+from common_constructs.user_pool import UserPool
+
+
+class AdminUsers(UserPool):
+    def __init__(
+            self, scope: Construct, construct_id: str, *,
+            cognito_domain_prefix: str,
+            environment_name: str,
+            encryption_key: IKey,
+            removal_policy,
+            **kwargs
+    ):
+        super().__init__(
+            scope, construct_id,
+            cognito_domain_prefix=cognito_domain_prefix,
+            environment_name=environment_name,
+            encryption_key=encryption_key,
+            removal_policy=removal_policy,
+            **kwargs
+        )
+
+        self._add_resource_server()
+        self.ui_client = self.add_ui_client(
+            ui_scopes=[
+                OAuthScope.OPENID,
+                OAuthScope.PROFILE,
+                OAuthScope.EMAIL,
+                OAuthScope.COGNITO_ADMIN,
+                OAuthScope.resource_server(self.resource_server, self.scope)
+            ]
+        )
+
+        # We will create some admins to get access started for the app and for support
```

Comment:
I thought it would be a good idea to pencil in the creation of an initial admin user for initial system set-up, but left it commented until we have proper environments set up. As to the timing, I'll leave that to @isabeleliassen and planning discussions.